spanner/cloud-client/src/main/java/com/example/spanner/LoadCsvExample.java

```diff
@@ -40,31 +42,40 @@
 
 /** Sample showing how to load CSV file data into Spanner */
 public class LoadCsvExample {
-
   public static final String EXCEL = "EXCEL";
   public static final String POSTGRESQL_CSV = "POSTGRESQL_CSV";
   public static final String POSTGRESQL_TEXT = "POSTGRESQL_TEXT}";
 
   enum SpannerDataTypes {
+    STRING,
     INT64,
     FLOAT64,
+    BYTE,
+    DOUBLE,
+    LONG,
     BOOL,
     DATE,
-    TIMESTAMP,
-    STRING
+    TIMESTAMP
   }
 
-  public static ArrayList<SpannerDataTypes> spannerSchema = new ArrayList<>();
-  public static ArrayList<String> headers = new ArrayList<>();
+  public static Boolean hasHeader = false;
   public static Connection connection;
+  public static Map<String, SpannerDataTypes> tableColumns = new LinkedHashMap<>();
 
+  /** Return the data type of the column type **/
   public static SpannerDataTypes parseSpannerDataType(String columnType) {
     if (columnType.matches("STRING(?:\\((?:MAX|[0-9]+)\\))?")) {
       return SpannerDataTypes.STRING;
-    } else if (columnType.equalsIgnoreCase("FLOAT64")) {
-      return SpannerDataTypes.FLOAT64;
     } else if (columnType.equalsIgnoreCase("INT64")) {
       return SpannerDataTypes.INT64;
+    } else if (columnType.equalsIgnoreCase("FLOAT64")) {
+      return SpannerDataTypes.FLOAT64;
+    } else if (columnType.equalsIgnoreCase("BYTE")) {
+      return SpannerDataTypes.BYTE;
+    } else if (columnType.equalsIgnoreCase("DOUBLE")) {
```

Comment:
I'm seeing both DOUBLE and FLOAT64 here, have you checked which types are supported? They are equivalent, I'm surprised to see both.

Reply:
I think Tina and I started a conversation about this in the Starter Project design doc if you want to continue the discussion over there.